presto-hive/src/main/java/com/facebook/presto/hive/metastore/SemiTransactionalHiveMetastore.java

```diff
@@ -319,9 +318,6 @@ public synchronized void truncateUnpartitionedTable(ConnectorSession session, St
         if (!table.isPresent()) {
             throw new TableNotFoundException(schemaTableName);
         }
-        if (!table.get().getTableType().equals(TableType.MANAGED_TABLE.toString())) {
```

Comment:
We should leave this check here. It's a good safety check since we never want to physically delete data for a non-managed table.

This entire commit can simply be removed.